edx_proctoring/tests/utils.py

```diff
@@ -373,32 +373,3 @@ def _normalize_whitespace(string):
         Replaces newlines and multiple spaces with a single space.
         """
         return ' '.join(string.replace('\n', '').split())
-
-
-def create_onboarding_exam(
```

Comment:
Removed this util function as it duplicated another and was causing discrepancies in tests.